env.yml

```diff
@@ -36,6 +36,7 @@ dependencies:
   - pytorch >=1.10.2
   - scikit-learn
   - fcd_torch
+  - sentencepiece
```

Comment:
if possible can you check whether there are hidden dependencies for sentencepiece earlier version that could clash ?

Reply:
I am not familiar with `sentencepiece`, so maybe these are naive remarks, but: 

Does it make sense to add this dependency if it's only being used by a single model? Is the dependency "stand-alone" (no subdependencies) and thus easy to install? Is it popular enough that we expect other models to use it too? Is that why?

Reply:
We use `conda-forge`,  so having a quick look at the recipe for sentencepiece is enough. Here: https://github.com/conda-forge/sentencepiece-feedstock/blob/main/recipe/meta.yaml 

It's always worth checking how any new dependency works or conflict with existing ones. For example a new dependencies might require a version of package with could be incompatible with an existing dependency.

I don't see anything that would be a problem in this case.

Reply:
I posted my comment before seeing yours @cwognum.

The way I would approach it is to add the optional dependency on `pyproject.toml`  for `pip` and in the env file here. Then document the requirement to make the model work. 

In the conda recipe we don't need to add that dependency, as users will install that themselves.